reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,325 @@
+.. index::
+   single: Forms; Fields; DateIntervalType
+
+DateIntervalType Field Type
+===========================
+
+.. versionadded:: 3.2
+   The DateIntervalType field type was introduced in Symfony 3.2.
+
+This field type allows the user to modify data that represents a specific
+date interval. It can be rendered as a integer or text input or select tags.
```

Comment:
Actually, let's re-word everything. I realized I wasn't fully understanding *when* this field would
be useful at first (i was thinking "date range" at first, not duration).

```rst
This field allows the user to select an *interval* of time. For example, if you want to
allow the user to choose *how often* they receive a status email, they could use this
field to choose intervals like every "10 minutes" or "3 days".

The field can be rendered in a variety of different ways (see `widget`_) and can be configured to
give you a ``DateInterval`` object, an `ISO 8601`_ duration string (e.g. ``P1DT12H``)
or an array (see `input`_).
```